Dockerfile.build.template

```diff
@@ -19,20 +19,21 @@ RUN apt-get update \
 RUN mkdir -p rootfs-overlay && \
 	ln -s /lib rootfs-overlay/lib64
 
-COPY package.json /usr/src/app/
 
-# Install only the production modules
-RUN JOBS=MAX npm install --production --no-optional --unsafe-perm \
-	&& npm dedupe
 
+# Install only the production modules that have C extensions
+RUN JOBS=MAX npm install --production --no-optional --unsafe-perm sqlite3 mkfifo \
```

Comment:
Should we change our dependencies to have all the modules that get built in as `devDependencies` and then the remaining ones we still have to install separately be the only ones in `dependencies`, that way we can avoid having a special dependencies list here